network-api/env.default

```diff
@@ -16,3 +16,18 @@ BUILD_THROTTLE_SECONDS=900
 SOCIAL_AUTH_GOOGLE_OAUTH2_KEY=
 SOCIAL_AUTH_GOOGLE_OAUTH2_SECRET=
 SOCIAL_AUTH_LOGIN_REDIRECT_URL=http://localhost:5000/soc/complete/google-oauth2/
+CSP_DEFAULT_SRC='self'
+CSP_SCRIPT_SRC='self' 'sha256-iN7wpJdxHlpujRppkOA8N0+Mzp0ZqZr3lCtxM00Y63c=' cdn.optimizely.com https://www.google-analytics.com/analytics.js
```

Comment:
The Django admin interface relies heavily on inline scripts and CSS, so we'll have to add that here and to `script-src`. (remove the sha source or browsers will ignore unsafe-inline)